packages/MSBot/docs/add-services.md

```diff
@@ -0,0 +1,284 @@
+## Connecting Your Bot to a Service
+
```

Comment:
We need to have examples of the usage. It's hard to map the description of the options with what I see in the Azure portal. For example, if I am using CosmosDB, what does service name mean and how can I find it in the Azure portal?

Reply:
Filed a bug to track this separately. https://github.com/Microsoft/botbuilder-tools/issues/576